dags/vz_populate_cr3_metadata.py

```diff
@@ -0,0 +1,81 @@
+import os
+from pendulum import datetime, duration
+from airflow.decorators import dag, task
+from airflow.operators.docker_operator import DockerOperator
+from utils.slack_operator import task_fail_slack_alert
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+REQUIRED_SECRETS = {
+    "AWS_BUCKET_NAME": {
+        "opitem": "Vision Zero CR3 Metadata Extraction",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Bucket for CR3s",
+    },
+    "AWS_BUCKET_ENVIRONMENT": {
+        "opitem": "Vision Zero CR3 Metadata Extraction",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS S3 Path for CR3s",
+    },
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Key",
+    },
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Access key",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Secret key",
+    },
+}
+
+
+@dag(
+    dag_id="vz-populate-cr3-metadata",
+    description="A DAG which populates CR3 metadata in the VZDB",
+    schedule_interval="*/5 8-10 * * *"
```

Comment:
It took me a second to realize that this schedule works together with the script processing 100 records per run. I can never remember why this is scheduled this way, and I have asked before so maybe a comment about it would be good (if we know why).

Reply:
I dont remember why it is scheduled this way either. maybe we dont have to do it this way anymore? (future refinement)